kernel.py

```diff
@@ -223,7 +234,8 @@ def _start_kernel(self):
         self._setup_streams()
         self._create_kernel()
 
-        self._logger.info("IPython: Start kernel now. pid: %i, thread: %r" % (os.getpid(), threading.current_thread()))
+        self._logger.info("IPython: Start kernel now. pid: %i, thread: %r",
+                          os.getpid(), threading.current_thread())
```

Comment:
Why this change?
Also, better do not indent like this. The indent should not be so long.

Reply:
This change (`+` -> `,`) was advice by my linter to get some lazy string concatenation.
I agree with the pending indent.
And as sparse is better than dense, I propose that: 99f148334

Reply:
Yes, that's ok.